app/src/main/java/org/mozilla/rocket/privately/PrivateModeActivity.kt

```diff
@@ -61,6 +72,15 @@ class PrivateModeActivity : LocaleAwareAppCompatActivity(), FragmentListener {
         }
     }
 
+    override fun getTabsSession(): TabsSession {
+        if (session == null) {
+            session = TabsSession(PrivateTabViewProvider(this))
+        }
+
+        // we just created it, it definitely not null
```

Comment:
it seems this method can be removed by making seesion a lazy property?

Reply:
if we assign `lateinit ` to it, Kotlin will complain on `if (session == null)`

Reply:
What I''m talking about is lazy delegate like
val session: TabSessions by lazy { TabSessions(...) }